app/managing-users.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Managing users
+analytics_key: managing-users
+parent:
+  title: How to use the pilot service
+  url: /guide
+previous:
+  title: Reporting
+  url: /reporting
+next:
+  title: Managing vaccines
+  url: /managing-vaccines
+---
+
+{% include 'guide-contents.njk' %}
+
+## Adding new users  
+
+Only lead administrators can add and manage users. 
+
+To add a user, you must enter their: 
+
+* First name  
+* Last name 
+* NHS-approved email address  
+* Clinician status
+
+## Selecting clinician status 
+
+Select ‘yes’ if users are registered clinicians. This will allow them to assess patients and record their consent.  
```

Comment:
I wonder if this is slightly confusing, as it’s technically about whether they can be recorded as having assessed the patient, rather than whether they're allowed to? Although that's quite a hard thing to explain. Hmm.

Reply:
Good point, instead 'Select ‘yes’ if users are registered clinicians. As a result, they can be selected as assessing and consenting clinicians.

Reply:
Nice. Have amended this.